schema/generate-schema.sh

```diff
@@ -10,7 +10,7 @@ if [ ! -x "${GOBIN}/schematyper" ]; then
 fi
 
 schema_version="v1"
-schema_json="../src/schema/${schema_version}.json"
+schema_json="../schema/${schema_version}.json"
```

Comment:
is this the correct file to point to? there is also a `src/v1.json`

Reply:
@darkmuggle if you could check the schema changes in general that would be great. i am not sure if that is all there is to be changed.